src/components/guess-count.js

```diff
@@ -3,9 +3,12 @@ import React from 'react';
 import './guess-count.css';
 
 export default function GuessCount(props) {
-    return (
-        <p>
-            Guess #<span id="count">{props.count}</span>!
-        </p>
-    );
+  const isPlural = props.guessCount !== 1;
+  const guessNoun = isPlural ? 'guesses' : 'guess';
+
+  return (
+    <h2 id="guessCount">
+      You&apos;ve made <span id="count">{props.guessCount}</span> {guessNoun}!
```

Comment:
Why do we need &apos; here?  Can we not just use an actual apostrophe?

Reply:
Unless v16 got less picky, the app warns about inescaped entity characters placed in the render function.